src/NServiceBus.Core.Tests/NServiceBus.Core.Tests.csproj

```diff
@@ -317,6 +317,7 @@
     <Compile Include="Timeout\FakeMessageDispatcher.cs" />
     <Compile Include="Timeout\InMemoryTimeoutPersisterTests.cs" />
     <Compile Include="Timeout\When_fetching_timeouts_from_storage.cs" />
+    <Compile Include="Transports\MSMQ\QueuePermissionsTests.cs" />
```

Comment:
Move to https://github.com/Particular/NServiceBus/tree/develop/src/NServiceBus.Core.Tests/Msmq to be consistent?